server/autograder.py

```diff
@@ -30,34 +33,40 @@ def send_autograder(endpoint, data):
                                                          error_message))
         raise ValueError(error_message)
 
-def autograde_assignment(assignment, ag_assign_key, token, autopromotion=True):
-    """ Autograde all enrolled students for this assignment.
-    If ag_assign_key is 'test', the autograder will respond with 'OK' but not grade.
+def autograde_assignment(assignment):
+    """Autograde all enrolled students for this assignment.
 
-    @assignment: Assignment object.
-    @ag_assign_key: Autograder ID (from Autograder Dashboard)
-    @token: OK Access Token (from auth)
+    @assignment: Assignment object
     """
+    if not assignment.autograding_key:
+        raise ValueError('Assignment has no autograder key')
+
+    # Create an access token for this run
+    autograder_client = Client.query.get('autograder')
+    if not autograder_client:
+        raise ValueError('Autograder OAuth client does not exist')
```

Comment:
It seems to easy for us to just add the "autograder" client here. Why not make it easier for people on the first install?